sdk/servicelinker/arm-servicelinker/README.md

```diff
@@ -0,0 +1,109 @@
+[Package (NPM)](https://www.npmjs.com/package/@azure/arm-servicelinker) |
+[API reference documentation](https://docs.microsoft.com/javascript/api/@azure/arm-servicelinker) |
+[Samples](https://github.com/Azure-Samples/azure-samples-js-management)
+
+## Getting started
+
+### Currently supported environments
+
+- [LTS versions of Node.js](https://nodejs.org/about/releases/)
+- Latest versions of Safari, Chrome, Edge and Firefox.
+
+See our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
+
+### Prerequisites
+
+- An [Azure subscription][azure_sub].
+
+### Install the `@azure/arm-servicelinker` package
+
+Install the Azure ServiceLinkerManagement client library for JavaScript with `npm`:
+
+```bash
+npm install @azure/arm-servicelinker
+```
+
+### Create and authenticate a `ServiceLinkerManagementClient`
+
+To create a client object to access the Azure ServiceLinkerManagement API, you will need the `endpoint` of your Azure ServiceLinkerManagement resource and a `credential`. The Azure ServiceLinkerManagement client can use Azure Active Directory credentials to authenticate.
+You can find the endpoint for your Azure ServiceLinkerManagement resource in the [Azure Portal][azure_portal].
+
+You can authenticate with Azure Active Directory using a credential from the [@azure/identity][azure_identity] library or [an existing AAD Token](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/identity/identity/samples/AzureIdentityExamples.md#authenticating-with-a-pre-fetched-access-token).
+
+To use the [DefaultAzureCredential][defaultazurecredential] provider shown below, or other credential providers provided with the Azure SDK, please install the `@azure/identity` package:
+
+```bash
+npm install @azure/identity
+```
+
+You will also need to **register a new AAD application and grant access to Azure ServiceLinkerManagement** by assigning the suitable role to your service principal (note: roles such as `"Owner"` will not grant the necessary permissions).
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: `AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`.
+
+For more information about how to create an Azure AD Application check out [this guide](https://docs.microsoft.com/azure/active-directory/develop/howto-create-service-principal-portal).
+
+```javascript
+const { ServiceLinkerManagementClient } = require("@azure/arm-servicelinker");
+const { DefaultAzureCredential } = require("@azure/identity");
+// For client-side applications running in the browser, use InteractiveBrowserCredential instead of DefaultAzureCredential. See https://aka.ms/azsdk/js/identity/examples for more details.
+
+const subscriptionId = "00000000-0000-0000-0000-000000000000";
+const client = new ServiceLinkerManagementClient(new DefaultAzureCredential(), subscriptionId);
```

Comment:
There is no subscriptionId in client constructor, @qiaozha Do you think we need to fix the readme generation?

Reply:
yes. please file an issue in sample generation and remove the sample generation for servicelinker this time.

Reply:
@colawwj Could you correct the example manually in readme file this time?